scripts/tinybird/tinybird_upload.py

```diff
@@ -0,0 +1,214 @@
+import csv
```

Comment:
I think it might make sense to rename this script a bit (or the folder). It doesn't really say what we are uploading here... 😄